src/queryAnalysis/utils.go

```diff
@@ -5,52 +5,59 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
+	"github.com/newrelic/infra-integrations-sdk/v3/log"
+	"github.com/newrelic/nri-mssql/src/args"
+	"github.com/newrelic/nri-mssql/src/queryAnalysis/config"
 	"regexp"
 	"strconv"
+	"sync"
 
 	"github.com/jmoiron/sqlx"
 	"github.com/newrelic/infra-integrations-sdk/v3/data/metric"
 	"github.com/newrelic/infra-integrations-sdk/v3/integration"
 	"github.com/newrelic/nri-mssql/src/queryAnalysis/models"
 )
 
-//go:embed config/queries.json
+//go:embed config/queries.go
```

Comment:
queries.go is standard golang file only , so can import directly using packages instead of embeded